ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-release-pipeline.yml

```diff
@@ -162,38 +179,48 @@ spec:
         - name: dist_method
           value: "$(tasks.get-cert-project-related-data.results.dist_method)"
         - name: image
-          value: "$(params.registry)/$(params.image_namespace)/$(tasks.operator-validation.results.package_name)-index:$(tasks.operator-validation.results.bundle_version)"
+          value: *bundleIndexImage
         - name: is_latest
           value: "$(params.is_latest)"
         - name: bundle_version
           value: "$(tasks.operator-validation.results.bundle_version)"
 
-    # create or update container image
-    - name: update-container-image
+    # create container image
+    - name: create-container-image
       runAfter:
         - publish-to-ocp-registry
+        - inspect-image
       taskRef:
-        name: update-container-image
+        name: create-container-image
       params:
-        - name: cert_project_id
-          value: "$(tasks.certification-project-check.results.certification_project_id)"
+        - name: isv_pid
+          value: "$(tasks.get-cert-project-related-data.results.isv_pid)"
         - name: registry
           value: "$(params.registry)"
```

Comment:
It doesn't seem like the pipeline is passing the correct registry here. According to the pipeline param description it's the:
```
temporary registry used by Hosted pipeline to pass the image to Release pipeline.
```

Yet the description on the task param is:
```
certification project registry
```
It should be the latter and it's mostly hard coded as `registry.connect.redhat.com` from what I've seen across all our pyxis envs. This is the production public facing partner registry proxy. There may not even be working preprod environments. Please verify if we can just hard code this.